server/src/database/remoteStorage.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  S3Client,
+  ListBucketsCommand,
+  ListObjectsV2Command,
+  GetObjectCommand,
+  PutObjectCommand,
+  type ListBucketsCommandOutput,
+  type ListObjectsV2CommandOutput,
+  type GetObjectCommandOutput,
+  type PutObjectCommandOutput,
+} from '@aws-sdk/client-s3'
+
+const accountId = 'f3ce51ca8eb25deaf3375a5d74535a64'
+const accessKeyId = 'a43841e46ea626187011747d8c79ce49'
+const secretAccessKey = 'a6d1f9a5f4530655aea82b9c9b5831ab8fae18c0a1cf51594da603c71c23cba7'
```

Comment:
Short-lived key for now

Reply:
You really shouldn't have any of those keys, people steal them pretty fast...

Reply:
Disabled